ui/client/common/DateUtils.js

```diff
@@ -1,10 +1,8 @@
+import Moment from "moment";
+import {displayDateFormat} from "../config";
 
 export default {
-
-
-  format(dateTimeString) {
-    //TODO: something better?
-    return dateTimeString.replace("T", " | ").substring(0, 18)
-  }
-
+    format(dateTimeString) {
```

Comment:
do we really want this? I think our users will be a bit confused. There *must* be at least some way of showing absolute date...

Reply:
What would you say if the absolute date would be displayed "on mouse over" specified date?

Reply:
maybe we can try it...